src/OpenDB/src/swig/common/CMakeLists.txt

```diff
@@ -1,16 +1,20 @@
+include("openroad")
+
 add_library(opendb_swig_common
     swig_common.cpp
 )
 
 target_include_directories(opendb_swig_common
   PUBLIC
     ${PROJECT_SOURCE_DIR}/include
+    ${OPENROAD_HOME}/include
```

Comment:
This just feels wrong.   I looked more closely at swig_common.cpp and it is mostly leftovers from when opendb was standalone.  If you look at read_lef, there is one in swig_common.cpp and OpenRoad::readLef in OpenRoad.cc.  The later is what we use in OR and I think we should just drop the one from opendb.  At the application level we have the logger on-hand and don't have to mess around like this.

Would you move the command and we can drop this and separately cleanup swig_common.cpp.